### ISSUE TITLE
Add an open license

### ISSUE THREAD

**Issue by lumbric** [pditommaso/awesome-pipeline#194]
It would be awesome if the repository could be shared under a CC-BY license.

Do we need to ask all the 82 current contributors to agree, if we want to add a license or re-license it?

**Comment by remram44** [comment]
IANAL but if the contributors shared *facts*, that's not copyrightable.

**Comment by lumbric** [comment]
@remram44 good point! However, the collection of links is probably copyrightable, so I guess that means that the repository owner could simply add a license file?

**Comment by remram44** [comment]
Probably?
